public/app/features/explore/Table.tsx

```diff
@@ -44,7 +49,25 @@ export default class Table extends PureComponent<TableProps> {
       accessor: text,
       className: VALUE_REGEX.test(text) ? 'text-right' : '',
       show: text !== 'Time',
-      Cell: row => <span className={filterable ? 'link' : ''}>{row.value}</span>,
+      Cell: row => (
+        <div>
+          {row.value}
+          {filterable && (
+            <span className="filter-links">
+              <Tooltip content="Filter for value" placement="bottom">
```

Comment:
The tooltip gets in the way for me by appearing and disappearing quickly, so I can't even click on the icon. I would definitely prefer a simple element title here for native tooltips.

Reply:
@davkal, I also noticed this as I was making changes, but couldn't reliably reproduce it. :thinking:

Had a quick look at the `Tooltip` implementation which eventually led to Popper.js, and I couldn't immediately see anything wrong. I'll replace with a simple `title` attribute for now. Perhaps the flicker is a bug worth a look later.